script/glam/generate_glean_sql

```diff
@@ -70,10 +70,15 @@ function write_clients_daily_aggregates {
 function main() {
     cd "$(dirname "$0")/../.."
 
-    local project=${PROJECT:-moz-fx-data-shared-prod}
-    local product=${PRODUCT:-org_mozilla_fenix}
+    local project=${SRC_PROJECT:-moz-fx-data-shared-prod}
```

Comment:
What's the reason for the change in variable name?

Reply:
This is so it doesn't collide with `PRODUCT` in `run_glam_sql` which is the destination project.  Keeping it separate lets us set source and destination separately in the airflow task